src/containers/board/articleList/artticleList.tsx

```diff
@@ -1,24 +1,134 @@
+
+  const lastArticleRef = useCallback(
+    (node: HTMLDivElement) => {
+      if (isFetchingNextPage) return;
+      if (observerRef.current) observerRef.current.disconnect();
+
+      observerRef.current = new IntersectionObserver((entries) => {
+        if (entries[0].isIntersecting && hasNextPage) {
+          fetchNextPage();
+        }
+      });
+
+      if (node) observerRef.current.observe(node);
+    },
+    [isFetchingNextPage, fetchNextPage, hasNextPage],
+  );
+
+  const articles = data?.pages.flatMap((page) => page.list) ?? [];
+
+  const handleSortChange = (sortType: ArticleOrder) => {
+    setOrderBy(sortType);
+    handleOffDropdown();
+  };
+
   return (
-    <div className='max-w-desktop mx-auto my-auto flex flex-col gap-32'>
+    <div className='max-w-desktop h-auto overflow-hidden mx-auto my-auto flex flex-col gap-32'>
       <div className='flex items-center justify-between'>
-        <p className='text-lg font-medium tablet:text-xl  tablet:font-bold text-text-primary'>
+        <p className='text-lg font-medium tablet:text-xl tablet:font-bold text-text-primary'>
           게시글
         </p>
-        <div className='bg-background-secondary w-120 h-44 rounded-12 flex items-center justify-center text-sm tablet:text-md text-text-disabled'>
-          최신순
-        </div>
-        {/* TODO : 드롭다운 컴포넌트로 변경하기 */}
-        {}
+        <Dropdown onClose={handleOffDropdown}>
+          <Dropdown.Trigger onClick={handleToggleDropdown}>
+            <div className='bg-background-secondary text-text-primary font-normal w-130 h-44 px-14 py-10 rounded-12 flex items-center justify-between text-sm tablet:text-md '>
+              {orderBy === 'like' ? '좋아요 많은순' : '최신순'}
+              <IconToggleDown className='text-icon-primary' />
+            </div>
+          </Dropdown.Trigger>
+          <Dropdown.Menu isOpen={isOpen} className='w-130 absolute top-44'>
+            <Dropdown.List
+              onClick={() => handleSortChange('like')}
+              onClose={handleOffDropdown}
+            >
+              좋아요 많은순
+            </Dropdown.List>
+            <Dropdown.List
+              onClick={() => handleSortChange('recent')}
+              onClose={handleOffDropdown}
+            >
+              최신순
+            </Dropdown.List>
+          </Dropdown.Menu>
+        </Dropdown>
       </div>
 
       <div className='flex flex-col gap-24 desktop:grid desktop:grid-cols-2'>
-        <ArticleCard />
-        <ArticleCard />
-        <ArticleCard />
+        {articles.map((board, index) => {
+          if (articles.length === index + 1) {
+            return (
+              <div key={board.id} ref={lastArticleRef}>
+                <ArticleCard board={board} onDeleteSuccess={refetch} />
+              </div>
+            );
+          }
+          return (
+            <ArticleCard
+              key={board.id}
+              board={board}
+              onDeleteSuccess={refetch}
+            />
+          );
```

Comment:
p3) 무한스크롤 때문이면 
```suggestion
              <div key={board.id} ref={lastArticleRef}>
                <ArticleCard board={board} onDeleteSuccess={refetch} />
              </div>
```
이대로 써도 될 것 같아요
ref를 굳이 밑에서 안 없애도 됩니당